ios/RNTwitterKitViewManager.m

```diff
@@ -8,6 +8,12 @@
 
 @implementation RNTwitterKitViewManager
 
+// this module is supposed to run in the main queue
+- (dispatch_queue_t)methodQueue
```

Comment:
What's the reason for moving the execution of the view manager on the main queue? It is not apparent.

Reply:
Well, it is indeed not required at the moment. However, it will be, once the library exports constants. I added it to prevent future thread problems. Since you might need it in the future anyway, the run time model remains the same. But of course, you can remove it if you wish. Should’t hurt at the moment.